prow/cmd/deck/static/tide.html

```diff
@@ -12,11 +13,16 @@
     </head>
     <body>
         <header>
-            <a href="/"><h1>Tide Status</h1></a>
+            <ul>
+                <li><a href="https://github.com/kubernetes/test-infra" class="logo"><img src="/logo.svg" alt="kubernetes logo" class="logo"></img></a><a href="/"><h1>Prow Status</h1></a></li>
+                <li><a href="/plugin-help.html"><h1>Plugin Help</h1></a></li>
+                <li><a href="/tide.html" class="current"><h1>Tide Status</h1></a></li>
+            </ul>
         </header>
         <article>
             <div>
-                <h4>Github search queries that define all merge requirements:</h4>
+                <h4>Pull Requests must have all tests / GitHub statuses be passing / green to merge&nbsp;<span class="success">✓</span></h4>
```

Comment:
The double `/` is confusing.
`All Github status contexts must be green to merge.` is probably sufficient given the line that follows.

Reply:
Agreed